internals/secrethub/init.go

```diff
@@ -162,27 +162,32 @@ func (cmd *InitCommand) Run() error {
 
 		client, err := cmd.newClientWithSetupCode(setupCode)
 		if err != nil {
+			cmd.progressPrinter.Stop()
```

Comment:
Not something to fix now, but I think we would make our lives a lot easier if we'd just allow `progressPrinter.Stop()` to be called multiple times. Then we can just defer the stopping.